manipulation/src/functTest/resources/mongo-java-driver-no-overwrite/build.gradle

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:3.0.3'
+        classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:4.3.1'
+
+        // Scala plugins
+        classpath "com.adtran:scala-multiversion-plugin:1.0.35"
+        classpath "com.diffplug.spotless:spotless-plugin-gradle:3.27.1"
+    }
+}
+
+plugins {
+    // including this plugin directly instead of by an init script, which allows to use the freshly build version
+    id 'org.jboss.gm.manipulation'
+    id 'maven-publish'
+}
+allprojects {
+    apply plugin: 'org.jboss.gm.manipulation'
+}
+
+
+//////////////////////////////////////////
+// Common behavior                      //
+//////////////////////////////////////////
+
+ext {
+    configDir = new File(rootDir, 'config')
+    jnrUnixsocketVersion = '0.18'
+    nettyVersion = '4.1.43.Final'
+    snappyVersion = '1.1.4'
+    zstdVersion = '1.3.8-3'
+    mongoCryptVersion = '1.0.1'
+    gitVersion = "" // getGitVersion()
+}
+
+def configDir = ext.configDir
+def coreProjects = subprojects.findAll { !['util'].contains(it.name) }
+def javaProjects = subprojects.findAll { !it.name.contains('scala') }
+def scalaProjects = subprojects.findAll { it.name.contains('scala') }
+def javaMainProjects = javaProjects.findAll { !['util'].contains(it.name) }
+def javaCodeCheckedProjects = javaMainProjects.findAll { !['driver-benchmarks'].contains(it.name) }
+
+configure(coreProjects) {
+    apply plugin: 'idea'
+
+//     evaluationDependsOn(':util')
+    group = 'org.mongodb'
+    version = '4.1.0.temporary-redhat-00001'
```

Comment:
Changed from the original project so that the version did not change (4.1.0 -> 4.1.0.temporary-redhat-00001).